docs/slurk_multibots.rst

```diff
@@ -3,4 +3,90 @@
 Pairing up participants and running multiple rooms (and bots)
 =============================================================
 
-*Explain how to run the waiting room / pairup bot, and the multibot.*
+In the following steps you will learn how to let bots and human users enter the slurk waiting room where users will be paired up and put into a new task-specific chatroom along with bots. The bot handling this mechanism (moving human users and other bots from the waiting room to the taskroom) is the **pairup bot**. You can find it in the *sample_bots* directory.
+
+Again, we will start the chat server in development mode and assume that you will use the default values in the config.ini, serving slurk on http://127.0.0.1:5000.
+
+First, we need to log in the bots and then the users.
+
+**Step 1:** 
+
+Open a terminal window and start slurk:
+
+  ``python chat.py``
+
+**Step 2:**
+
+Go to http://127.0.0.1:5000/token and fill in the following:
+
+  - Source:  *can be anything*
```

Comment:
*can be anything* is only partly true as this fields exists for a reason. In this example, ofc, this holds. However in production, this is meant to identify e.g. different sessions.